src/data/community-events.json

```
@@ -45,6 +45,15 @@
     "endDate": "2023-04-16"
   },
   {
+    "title": "EDCON 2023 MONTENEGRO",
```

Please only use the actual name of the conference, not it's location.

```suggestion
    "title": "EDCON 2023",
```